code/datums/gamemodes/spy_theft.dm

```diff
@@ -721,17 +721,25 @@
 	possible_areas = get_areas_with_unblocked_turfs(/area/station)
 	possible_areas += get_areas_with_unblocked_turfs(/area/diner)
 	possible_areas -= get_areas_with_unblocked_turfs(/area/diner/tug)
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/medical/asylum)			// Donut 3 Asylum
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/security/hos)
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/security/armory)			// Cog1 armory
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/ai_monitored/armory)	// Other map armorys...
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/turret_protected)		// AI secure areas
 	possible_areas -= get_areas_with_unblocked_turfs(/area/station/maintenance)
 	possible_areas -= get_areas_with_unblocked_turfs(/area/station/hallway)
 	possible_areas -= get_areas_with_unblocked_turfs(/area/station/engine/substation)
+	possible_areas -= get_areas_with_unblocked_turfs(/area/station/engine/core)
```

Comment:
see HoS

Reply:
Right, should be ok. Sometimes heat is a problem but I suppose there are fire suits nearby.